docs/BR.md

```diff
@@ -1756,142 +1754,916 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| `countryName`            | MAY          | The two-letter ISO 3166-1 country code for the country associated with the Subject. | [Section 3.2.2.3](#3223-verification-of-country) |
+| `commonName`             | SHOULD NOT   | If present, MUST contain a single IP address or Fully-Qualified Domain Name that is one of the values contained in the Certificate's `subjectAltName` extension. | |
+| Any other attribute      | MUST NOT     | __**TBD**__ |                  |
+
+##### 7.1.2.6.3 Individual Validated
+
+For a Subscriber Certificate to be Individual Validated, it MUST meet the following profile:
+
+| __Field__             | __Requirements__     |
+| --                    | -------              |
+| `subject`             | See following table. |
+| `certificatePolicies` | MUST be present and MUST assert the [Reserved Certificate Policy Identifier](#7161-reserved-certificate-policy-identifiers) of `2.23.140.1.2.3` as a `policyIdentifier`. |
+| All other extensions  | See [Section 7.1.2.6.6](#71266-subscriber-certificate-extensions) |
+
+All `subject` names MUST be encoded as specified in [Section 7.1.4](#714-name-forms).
+
+The following table details the acceptable `AttributeType`s that may appear within the `type` field of an `AttributeTypeAndValue`, as well as the contents permitted within the `value` field.
+
+Table: Individual Validated `subject` Attributes
+
+| __Attribute Name__       | __Presence__ | __Value__   | __Verification__ |
+| ---                      | -            | ------      | --               |
+| `countryName`            | MUST         | The two-letter ISO 3166-1 country code for the country associated with the Subject. If a Country is not represented by an official ISO 3166-1 country code, the CA MUST specify the ISO 3166-1 user-assigned code of `XX`, indicating that an official ISO 3166-1 alpha-2 code has not been assigned. | [Section 3.2.3](#323-authentication-of-individual-identity) |
+| `stateOrProvinceName`    | MUST / MAY   | MUST be present if `localityName` is absent, MAY be present otherwise. If present, MUST contain the Subject's state or province information. | [Section 3.2.3](#323-authentication-of-individual-identity) |
+| `localityName`           | MUST / MAY   | MUST be present if `stateOrProvinceName` is absent, MAY be present otherwise. If present, MUST contain the Subject's locality information. | [Section 3.2.3](#323-authentication-of-individual-identity) |
+| `streetAddress`          | SHOULD NOT   | If present, MUST contain the Subject's street address information. | [Section 3.2.3](#323-authentication-of-individual-identity) |
+| `postalCode`             | SHOULD NOT   | If present, MUST contain the Subject's zip or postal information. | [Section 3.2.3](#323-authentication-of-individual-identity) |
+| `organizationName`       | SHOULD NOT   | If present, MUST contain the Subject's name or DBA. | [Section 3.2.3](#323-authentication-of-individual-identity) |
```

Comment:
The current Requirements allow for the Subject's name to be expressed in the O and/or GN + SN. It appears this is movement towards deprecating the O in favor of the GN + SN. However, it appears that current practice today is exclusive use of the O and omitting the GN + SN: https://censys.io/certificates/help?q=%28parsed.extensions.certificate_policies.id%3A+2.23.140.1.2.3+and+parsed.subject.given_name%3A+%2A%29+AND+tags.raw%3A+%22trusted%22

Given current practice, can you explain a bit on the rationale behind this change?

Reply:
Right, but this keeps it as SHOULD NOT; it reflects that these are not necessary browser use cases, that they should support the more strongly typed fields if that's important to them, but still permits CAs to issue them (i.e. it doesn't go full MUST NOT)

It sounds like you'd prefer MAY over SHOULD NOT, and while I'm not sure I agree, I'm just wanting to make sure I understand the feedback (i.e. that there's not a more glaring bug here and I'm just misunderstanding you)

Reply:
Thanks for clarifying. I believe *SHOULD NOT* remains appropriate with respect to RFC 2119 here:

> 4. SHOULD NOT   This phrase, or the phrase "NOT RECOMMENDED" mean that there may exist valid reasons in particular circumstances when the particular behavior is acceptable or even useful, but the full implications should be understood and the case carefully weighed before implementing any behavior described with this label.

The definition of MAY here is not consistent with your hypothetical use case, in particular:
> In the same vein an implementation which does include a particular option MUST be prepared to interoperate with another implementation which does not include the option (except, of course, for the feature the option provides.)

Your hypothetical software that requires it is inconsistent with the MAY usage. The SHOULD NOT, however, is consistent with the consideration that the full implications - in particular, field confusion - apply.